### ISSUE TITLE
[Feature] "The selected articles have been marked as read." notification is shown for too long.

### ISSUE THREAD

**Issue by rezad1393** [marienfressinaud/FreshRSS#4170]
**Is your feature request related to a problem? Please describe.**
I use mark as read a lot of times.
when I do that a notification inside freshrss interface is show as  "The selected articles have been marked as read." that is shown for 5 seconds straight.
this is not needed for (at least for that long)
can I disable this UI item or at least reduce its display time?

**Comment by math-GH** [comment]
Right now it is not possible to configure the display time.

What disadvantages do you have right now with a 5 seconds display time?

Do you know, that you can click on the banner (mobile view) or on the `x` right hand side of the banner (desktop view) and the banner will be closed?

**Comment by rezad1393** [comment]
yes I understand this is not hearthblead level of security severity. :)
just a papercut kinda-annoying.
as I said I use that button a lot so I would have to close it many times and I really don't need that notification .

**Comment by math-GH** [comment]
With CustomCSS extension you can hide all "green" messages. 
The CSS classes for that message are: `notification` and `good`

**Comment by rezad1393** [comment]
> With CustomCSS extension you can hide all "green" messages. The CSS classes for that message are: `notification` and `good`

that may hide other messages too.
is there a more specific one?
I am not familiar with css, so If you can find a precise one can you paste that here?

**Comment by math-GH** [comment]
It would hide all "green" messages. There is no more specific one.

**Comment by kmac** [comment]
My problem with this is that it also blocks the top-level action buttons.  So if I'm powering through some feeds, I have to wait 5s to click on the next Mark As Read button.

If the notification was moved to one of the corners (preferably bottom) that would be an improvement.

(this is definitely a minor issue - but I was looking for a way to control this behaviour and didn't find anything)

**Comment by math-GH** [comment]
Have you checked out the other themes? F.e. the Swage theme displays the notifications not at the top
